backend/integration_tests/src/account_linking_tests.rs

```diff
@@ -3,7 +3,8 @@ use crate::env::ENV;
 use crate::utils::now_millis;
 use crate::{TestEnv, client};
 use identity_canister::create_account_linking_code::Response as CreateAccountLinkingCodeResponse;
-use identity_canister::link_with_account_linking_code::Response as LinkWithAccountLinkingCodeResponse;
+use identity_canister::finalise_account_linking_with_code::Response as FinaliseAccountLinkingWithCodeResponse;
+use identity_canister::verify_account_linking_code::Response as VerifyAccountLinkingCodeResponse;
 use std::ops::Deref;
 use testing::rng::random_delegated_principal;
```

Comment:
On line 17 you should use `random_principal` to mimic the random temp key that each user will generate.